readme.md

```diff
@@ -9,6 +9,14 @@ The ineptly named tool which may one day supercede the older [build tools](https
 
 ## Install
 
+### Docker
+
+Assuming you have docker, docker-compose and bash installed, you don't need any extra steps and can use caw straight away through `docker/cow` script. You can use it from any other place on your drive - it will automatically mount the current folder as the working directory.
```

Comment:
```suggestion
Assuming you have docker, docker-compose and bash installed, you don't need any extra steps and can use cow straight away through `docker/cow` script. You can use it from any other place on your drive - it will automatically mount the current folder as the working directory.
```